scripts/update-tools

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -ueo pipefail
+dominikh="$(curl https://api.github.com/repos/dominikh/go-tools/releases/latest -s | jq -r .tag_name)"
+sed -i'' "s/staticcheck@.*$/staticcheck@$dominikh/" Dockerfile.jenkins
+
+golangci="$(curl https://api.github.com/repos/golangci/golangci-lint/releases -s | jq -r '.[].tag_name' | sort -rV | head -n 1)"
```

Comment:
note, makefile contains also golangci-lint

Reply:
I was going to comment on this same thing and ask @brimstone to apply its sed magic to Makefile if that it doesn't require that much time.